pages/dashboard.vue

```diff
@@ -44,8 +44,8 @@
                 online videos, and downloadable quick guides.
               </p>
               <v-row no-gutters class="align-bottom pt-7">
-                <span class="primary--text font-weight-bold">Go to the Registry Application Help Page</span>
-                <v-icon color="primary" small right>mdi-open-in-new</v-icon>
+                <span class="font-weight-bold" style="color: #1669BB">Go to the Registry Application Help Page</span>
+                <v-icon color="#1669BB" small right>mdi-open-in-new</v-icon>
```

Comment:
Are these colours defined in main.ts? or in the assets theme.scss?
If not, we can do so and apply them here vs hex colours in the templates. 

Also worth verifying the font size, is the default larger than 14px?

Reply:
1. it's in theme.scss as $primary-blue, but not in main.ts. In main.ts, primary is defined as "#1a5a96"
2. the default was 16px

Reply:
Fair enough. 
You could take this opportunity to define this in main.ts if you like. 
See https://github.com/bcgov/ppr/blob/main/ppr-ui/src/main.ts

Reply:
oh i see, so primary is already defined in main.ts?
Seems a bit out of sync with our Registry theme

Reply:
Or maybe ppr is the one out of sync?

Reply:
An interesting take but that would mean, the Edit UI, Create UI, Name Request UI and Filings UI (All registries projects) are out of sync as well ;) 

Please confirm this with UX and update the `primary` colour definition if so.

Reply:
just talked to Tracey, and yes the colors are intended to be different.

Reply:
Sorry, to clarify, you mean the primary colour is intended to be different?

Reply:
If so, all good. 
That being said, this blue is consistently used throughout the apps in the themes. 
It would be recommended if you could define it on the Vuetify instance for reuse in the future and to help avoid the use of hex colours in the templates. 👍

Reply:
Yes Primary is intended to be different.

Ok I'll add a color for it.

Reply:
Thanks for the updates Gary 👍 
Working on getting write access here so i can assist further on moving these pr's and resolving comments etc